hepcrawl/extractors/aps_parser.py

```diff
@@ -32,11 +35,16 @@ def parse(self, response):
             record = HEPLoader(item=HEPRecord(), response=response)
 
             journal_doctype = self.article_type_mapping.get(article.get('articleType'), 'other')
+            if journal_doctype == 'other':
+                logger.warning('Journal_doctype is %s. Do we need other mapping for this?' % journal_doctype)
+
             record.add_value('journal_doctype', journal_doctype)
             record.add_value('dois', get_nested(article, 'identifiers', 'doi'))
             record.add_value('page_nr', str(article.get('numPages', '')))
 
             arxiv = get_nested(article, 'identifiers', 'arxiv').replace('arXiv:', '')
+            if not arxiv:
```

Comment:
so, if `arxiv` is `None` or an empty string we still create the `arxiv_eprints` object with an empty value?